code/__HELPERS/shell.dm

```diff
@@ -0,0 +1,56 @@
+//Runs the command in the system's shell, returns a list of (error code, stdout, stderr)
+
+#define SHELLEO_NAME "data/shelleo."
+#define SHELLEO_ERR ".err"
+#define SHELLEO_OUT ".out"
+/proc/shelleo(command)
```

Comment:
/world/proc

/proc can be admin called and abused

Reply:
yikes!